README.md

```diff
@@ -765,6 +766,105 @@ NB: Using the JSON Messsage Generator option it is possible to format messages d
 
   More info and an example can be found [here](https://aws.amazon.com/blogs/compute/continuous-integration-deployment-for-aws-lambda-functions-with-jenkins-and-grunt-part-1/)
 
+### Trigger a Lambda function when an email is received by Amazon Simple Email Service
+
+Start by creating a blank AWS Lambda function that will be called whenever a new email comes in.
+
+#### Set up SES
+For this you need your own domain, and you need to verify the domain with AWS, this can take up to 72hrs, so have a nice drink and chill out while you wait :sunglasses: :coffee:
+
+	See here for how: http://docs.aws.amazon.com/ses/latest/DeveloperGuide/verify-domain-procedure.html
+
+#### Add a rule set
+Click `rule sets` on the bottom left and create a new rule set with whatever name you like. If you want to have your function triggered by emails sent to any address in your domain you can leave recipient blank, else you can add the recipient you want to trigger the function.
+
+In that rule set add an action for your Lambda function. This tells AWS to run your action when an email is received by SES.
+
+This is all you need to do to make your function trigger from SES, but depending on what your Lambda function does, it might be kind of hard to verify that it's run (though you can look at the stats for the function to see how and when it's been run) so let's make it save the body of your email to your bucket!
+
+#### Save the email to S3
+Go back to the rule set you created for your Lambda function. Add a new action to it, this one should be an S3 action, with the bucket you want to use selected. This will save the email to S3. Make sure this action is positioned **above** your Lambda function:
+
+![ses management console - google chrome_006](https://user-images.githubusercontent.com/22300773/28177094-a0e16bfc-67f1-11e7-8676-feabc437295f.png)
+
+This saves the email in a slightly weird way that we don't have that much control over, so we want our Lambda function to take that go into the file written from the S3 action and process it separately.
+
+So now you need to set up a role to give your function access to S3, for the policy of the role put something like this:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "logs:CreateLogGroup",
+            "Resource": "arn:aws:logs:us-east-2:YOUR-USER-ID-HERE:*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "logs:CreateLogStream",
+                "logs:PutLogEvents",
+                "s3:*" // This allows all S3 functions
```

Comment:
fairly certain these comments aren't allowed. (_which is a shame because they are instructive ..._)